editor/src/components/canvas/canvas-strategies/strategies/absolute-duplicate-strategy.tsx

```diff
@@ -165,10 +143,27 @@ function isApplicable(
       EP.parentPath(element),
     )
 
+    const isElementContentAffecting = treatElementAsContentAffecting(
+      canvasState.startingMetadata,
+      canvasState.startingAllElementProps,
+      element,
+    )
+
+    const isElementAbsolute = isElementContentAffecting
+      ? MetadataUtils.getChildrenPathsUnordered(canvasState.startingMetadata, element).every(
```

Comment:
this should be recursively unrolling the children instead! `replaceContentAffectingPathsWithTheirChildrenRecursive` to the rescue